app/routes/circulars/circulars.server.ts

```diff
@@ -379,9 +383,30 @@ export async function putVersion(
     editedBy: formatAuthor(user),
     editedOn: Date.now(),
   }
+
   validateCircular(newCircularVersion)
 
-  return await circularVersionsAutoIncrement.put(newCircularVersion)
+  const promises = [
+    circularVersionsAutoIncrement.put(newCircularVersion),
+  ] as any[]
```

Comment:
Please try to avoid using type assertions, because they can mask bugs that type inference and type checking would reveal.